modules/virt-configuring-live-migration-limits.adoc

```diff
@@ -6,19 +6,21 @@
 [id="virt-configuring-live-migration-limits_{context}"]
 = Configuring live migration limits and timeouts
 
-Configure live migration limits and timeouts for the cluster by adding updated 
-key:value fields to the `kubevirt-config` configuration file, which is located in the 
+Configure live migration limits and timeouts for the cluster by adding updated
+key:value fields to the `kubevirt-config` configuration file, which is located in the
 `openshift-cnv` namespace.
 
 .Procedure
 
-* Edit the `kubevirt-config` configuration file and add the necessary 
+* Edit the `kubevirt-config` configuration file and add the necessary
 live migration parameters. The following example shows the default values:
 +
+[source, terminal]
 ----
 $ oc edit configmap kubevirt-config -n openshift-cnv
 ----
 +
+.Example output
```

Comment:
Same here. The `oc edit` command just opens the file in a text editor.

Reply:
Done. I don't really understand this differentiation though.